### DISCUSSION TITLE
Wineskin Winery installation from scratch

### DISCUSSION THREAD

**Discussion by nmt1900** [The-Wineskin-Project/WineskinServer#579] [Q&A]
I have been using Wineskin Winery for many years for few work-related Windows apps, but now I am ending up in a position where I do not know how could I do new installation because straight-out available Wineskin Winery app download seems to be of version 1.8.4.2 from 2020 and it does have issues with new engines.

Current install instructions lead to Homebrew, which could be just OK - except that I am using Macports so getting Homebrew for this single application seems to be questionable considering issues with co-existense of Homebrew and Macports.

Have I got something wrong here or is getting Homebrew only viable way forward?

**Comment by Gcenx** [comment]
If you use macports you can add the following Portfile to your local overlay https://github.com/Gcenx/macports-wine/tree/main/emulators/wineskin

I might submit that to macports-ports directly but currently undecided as I’m about to take maintainer ship of a number of other ports.

**Reply by Gcenx** [reply]
I personally primarily use macports, but have found it useful to also have brew available for casks.

See https://docs.brew.sh/Installation#alternative-installs

Doing that has allowed me to keep using macports for everything and still having access to brew for casks.

**Comment by nmt1900** [comment]
I added portfile to macports sources and it looks like local repository is recognized - but macports does not find any new packages related to that. Am I missing something?
